crates/release_plz_core/src/command/release.rs

```diff
@@ -1000,6 +1001,26 @@ mod tests {
         static ref NO_PARALLEL: Mutex<()> = Mutex::default();
     }
 
+    fn with_env_var<K, V, F>(key: K, value: V, f: F)
+    where
+        K: AsRef<OsStr>,
+        V: AsRef<OsStr>,
+        F: FnOnce(),
+    {
+        // Store the previous value of the var, if defined.
+        let previous_val = env::var(key.as_ref()).ok();
+
+        env::set_var(key.as_ref(), value.as_ref());
+        (f)();
+
+        // Reset or clear the var after the test.
```

Comment:
this doesn't work because each test doesn't run in an isolated process. So this env var will be shared with other tests that run at the same time.